packages/re6stnet.rb

```diff
@@ -0,0 +1,39 @@
+require 'package'
+class Re6stnet < Package
+  description 'Resilient, Scalable, IPv6 Network'
+  homepage 'https://re6st.nexedi.com/'
+  version '0.501'
+  source_url 'https://lab.nexedi.com/nexedi/chromebrew-buildout-re6st/repository/archive?sha=master'
+  source_sha256 '056a94304282f9e073b85370be0b77c5f42a16fe0365602e921679d1287137fa'
+
+  binary_url ({
+    x86_64: 'https://softinst62285.host.vifib.net/public/re6stnet-0.501-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    x86_64: '39e86454e8e11ba9672145a2fa5ab4754c1c739a9062f2019e6a2f066787e064',
+  })
+
+  depends_on 'python27' => :build
+  depends_on 'filecmd' => :build
+
+  @re6st_dir = 	"#{CREW_PREFIX}/opt/re6st"
+
+  def self.build
+    system "mkdir -p #{@re6st_dir}/bin && cp buildout.cfg #{@re6st_dir} && cp bin/grandenet #{@re6st_dir}/bin"
+    system "SSL_CERT_DIR=/etc/ssl/certs ./bin/buildout buildout:directory=#{@re6st_dir}"
```

Comment:
While it's definitely possible, we want the binaries to be located in `#{@re6st_dir}` but also available on $PATH; would it be better to add the folder to $PATH rather than symlinking?

Reply:
I also received this error while attempting to build glib on i686:
```
  CC       libglib_2_0_la-gbytes.lo
  CC       libglib_2_0_la-gcharset.lo
  CC       libglib_2_0_la-gchecksum.lo
  CC       libglib_2_0_la-gconvert.lo
gconvert.c:59:2: error: #error GNU libiconv not in use but included iconv.h is from libiconv
 #error GNU libiconv not in use but included iconv.h is from libiconv
  ^~~~~
make[4]: *** [Makefile:1828: libglib_2_0_la-gconvert.lo] Error 1
make[4]: Leaving directory '/usr/local/opt/re6st/parts/glib__compile__/glib-2.50.2/glib'
make[3]: *** [Makefile:2508: all-recursive] Error 1
make[3]: Leaving directory '/usr/local/opt/re6st/parts/glib__compile__/glib-2.50.2/glib'
make[2]: *** [Makefile:1223: all] Error 2
make[2]: Leaving directory '/usr/local/opt/re6st/parts/glib__compile__/glib-2.50.2/glib'
make[1]: *** [Makefile:1213: all-recursive] Error 1
make[1]: Leaving directory '/usr/local/opt/re6st/parts/glib__compile__/glib-2.50.2'
make: *** [Makefile:855: all] Error 2
glib: Command failed with exit code 2: make 
glib: Compilation error. The package is left as is at /usr/local/opt/re6st/parts/glib__compile__/glib-2.50.2 where you can inspect what went wrong
While:
  Installing glib.
Error: System error
re6stnet failed to install: `SSL_CERT_DIR=/etc/ssl/certs ./bin/buildout buildout:directory=/usr/local/opt/re6st` exited with 1
```

Reply:
I realise this is not ideal, but this buildout recipe is what we use for our Debian distribution of re6st, and it would be hard for us to maintain two different versions over time. And we can't just use the chromebrew version for all packages, because python and openssl cause some issues

Sorry, but I don't have an i686 board to test  this on, but do you have the libiconv chromebrew package installed?

Reply:
@lzmartinico: `crew install libiconv` is all you need.  If you know the specifics of the edge cases, we could incorporate those changes into the crew packages to accommodate.